packages/pwa-kit-runtime/src/utils/ssr-config.server.js

```diff
@@ -28,21 +31,25 @@ const SUPPORTED_FILE_TYPES = ['js', 'yml', 'yaml', 'json']
  * Each file marked with `ext` can optionally be terminated with `js`, `yml|yaml` or
  * `json`. The file loaded is also determined based on that precidence of file extension.
  *
+ * @param {Object} opts.buildDirectory - Option path to the folder containing the configution. Byt default
+ * it is the `build` folder when running remotely and the project folder when developing locally.
  * @returns - the application configuration object.
  */
 /* istanbul ignore next */
-export const getConfig = () => {
-    const isRemote = Object.prototype.hasOwnProperty.call(process.env, 'AWS_LAMBDA_FUNCTION_NAME')
+export const getConfig = (opts = {}) => {
+    const {buildDirectory} = opts
     let targetName = process?.env?.DEPLOY_TARGET || ''
 
     const targetSearchPlaces = SUPPORTED_FILE_TYPES.map((ext) => `config/${targetName}.${ext}`)
     const localeSearchPlaces = SUPPORTED_FILE_TYPES.map((ext) => `config/local.${ext}`)
     const defaultSearchPlaces = SUPPORTED_FILE_TYPES.map((ext) => `config/default.${ext}`)
 
+    const searchFrom = buildDirectory || process.cwd() + '/' + CONFIG_SEARCH_DIRECTORY
+
     // Combined search places.
     const searchPlaces = [
-        ...targetSearchPlaces,
-        ...(!isRemote ? localeSearchPlaces : []),
+        ...(targetName ? targetSearchPlaces : []),
```

Comment:
`targetName` is potentially undefined as mentioned in a previous comment. This condition ensures we aren't searching for files that we shouldn't. This condition could be places above, but I put it here to go with out the other condition below it works.